.github/workflows/pull-request.yml

```
@@ -57,10 +57,10 @@ jobs:
 
       - name: Check for linting errors in modified files
         if: steps.changed-files.outputs.only_changed != 'true'
-        env: 
+        env:
           CHANGED_FILES: ${{ steps.changed_files.outputs.all_changed_files }}
```

actionlint

property "changed_files" is not defined in object type {changed-files: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)